dev/loadgen/prod-benchmark.yaml

```diff
@@ -1,14 +1,15 @@
 ## start with
 ##    loadgen benchmark prod-benchmark.yaml
 
-workspaces: 10
+workspaces: 100
 ideImage: eu.gcr.io/gitpod-core-dev/build/ide/code:commit-ff263e14024f00d0ed78386b4417dfa6bcd4ae2f
 waitForRunning: "600s"
 waitForStopping: "600s"
-successRate: 0.80
+successRate: 0.95
```

Comment:
Honest question, why not 100%? 

What does that rationale or use case look like, when it is >= 95, but, < 100?